CHANGELOG.md

```diff
@@ -9,5 +9,6 @@
 ### Breaking Changes
 
 ### Additions and Improvements
+- New attestation pool improving attestation packing during block production and aggregation. It is possible to revert to previous implementation via `--Xaggregating-attestation-pool-v2-enabled=false` if any issue is encountered.
```

Comment:
I find this not very reassuring/clear. Maybe rephrase it to something like:
- Enabling, by default, a new attestation pool implementation that improves the attestations packing during block production. It can still be disabled by setting `--Xaggregating-attestation-pool-v2-enabled=false` if needed

Reply:
fair :)

Reply:
changed a bit